### ISSUE TITLE
service.Tags.GetRecentMedia("tag_name") changed?

### ISSUE THREAD

**Issue by ma2c** [abjerner/Skybrud.Social#49]
Hi,

I coded a script to get recent media that contains a specific tag. It was working perfectly.

Unfortunately, some days ago I realized that the script is returning only recent media from the user account (the one that provided the access token).

Is there any change on Instagram API or Skybrud regarding to getting recent media? I search on Instagram Developers website and didn't find any news.

The steps I'm following to replicate the problem are:

1- Create a new service using an access token
2- Getting recent media by using -----> media = service.Tags.GetRecentMedia("tag");
3- The result is that I only get media from the user that created the service (previously I was getting ALL recent media from ANY user following exactly the same steps)

Any idea about why I'm only getting recent media from one user and not any user?

Thanks!

Mac.

**Comment by abjerner** [comment]
Hi Mac,

This is probably due to a recent change by Instagram, although I can't replicate your issue the way you describe it.

Instagram has recently updated their guidelines, which automatically affects new apps registered after the 17th of November. The new guidelines restricts access to content that is not your own - eg. users should now grant access to the `public_content` scope.

However if your app was registered before this change (even if you authenticated at a later date), it should still work as before. I just confirmed this with one of my own apps.

If your app is created after this date, you should grant the `public_content` scope. By not doing so, and trying to call `GetRecentMedia` in the tags endpoint, I simply get the following exception: `This request requires scope=public_content, but this access token is not authorized with this scope. The user must re-authorize your application with scope=public_content to be granted this permissions.`

Since my test results are a bit different from what you're experiencing, I'm not sure about the exact cause to your issue. But you can read more about the Instagram update here: 

http://developers.instagram.com/post/133424514006/instagram-platform-update

Also, I haven't yet had the time to read up in the updated API documentation, so you may also be able to find some answers there.

**Comment by ma2c** [comment]
Hi!

Thanks for your support.

I realized that the problem is due to Sandbox. I removed my previous app and created a new app and now I have to apply to Sandbox's approval.

It still works as before but since I'm the only user in my new app's Sandbox service then Tags.GetRecentMedia only retrieves my media and not all user's. I confirmed this when I added another user to my Sandbox, then I get media from both users.

Thank you for your help!


Mac.
